docs/ossm/istioctl/README.md

```diff
@@ -0,0 +1,70 @@
+[Return to OSSM Docs](../)
+
+# Installing the istioctl tool
+
+The `istioctl` tool is a configuration command line utility that allows service 
+operators to debug and diagnose Istio service mesh deployments.
+
+## Prerequisites
+
+* Use an `istioctl` version that is the same version as the Istio control plane 
+for the Service Mesh deployment. For checking the `Istio` version, run the following command:
+
+    ```bash
+    $ kubectl -n istio-system get istio
+    ```
+
+## Steps
+
+1. Download `istioctl` binary
+
+    In the OpenShift console, navigate to the Command Line Tools by clicking :grey_question: -> **Command Line Tools** in the upper-right of the header.  
+    Then click on **Download istioctl** and choose the right architecture according to your system.
+
+    **NB**: All the releases of `istioctl` are directly downloadable [here](https://mirror.openshift.com/pub/cgw/servicemesh/)
+
+1. Extract the `istioctl` binary and add the client to your path, on your system.
+
+    ```bash
+    $ tar xzf istioctl-<OS>-<ARCH>.tar.gz -C $HOME/.istioctl/bin
+    $ export PATH=$HOME/.istioctl/bin:$PATH
+    ```
+
+1. Confirm that the `istioctl` client version and the Istio control plane 
+version now match (or are within one version) by running the following command
+at the terminal:
+  
+    ```sh
+    $ istioctl version
+    ```
+
+## Supported commands
+
+|Command                | Description                                                                            | Supported          |
```

Comment:
How was this list of supported commands created?

Reply:
From feedback of the team but there might be some gaps

Reply:
@dgn @sridhargaddam @jewertow @yxun @jwendell @luksa @nrfox could you please take a look of the commands we should support and confirm that the list is correct.
Thank you